relay-server/src/utils/unreal.rs

```diff
@@ -83,7 +83,7 @@ pub fn expand_unreal_envelope(
 
         let mut item = Item::new(ItemType::Attachment);
         item.set_filename(file.name());
-        item.set_payload(content_type, file.data());
+        item.set_payload(content_type, file.data().to_owned());
```

Comment:
Does this make a copy of the data? Can we move it in somehow instead?

Reply:
Yes, this requires an upstream change in Symbolic. Adding a TODO for this and similar cases.